treadmill/infra/setup/freeipa.py

```diff
@@ -0,0 +1,18 @@
+from treadmill.infra.instances import Instances
+
+
+class FreeIPA:
+    def __init__(self):
+        self.instance = None
+
+    def setup(self, SubnetId=''):
+        self.instances = Instances.create(
+            Name='TreadmillFreeIPA',
+            ImageId='ami-9e2f0988',
```

Comment:
Default - for cli, and this is so obscure, better make it constant somewhere. i do not know what ami-9e.. means.